src/Raven.Studio/typescript/viewmodels/database/settings/databaseRecord.ts

```diff
@@ -68,6 +68,15 @@ class databaseRecord extends viewModelBase {
         const editorElement = $("#dbDocEditor");
         if (editorElement.length > 0) {
             this.docEditor = ko.utils.domData.get(editorElement[0], "aceEditor");
+
+            let alreadyFolded = false;
+            
+            this.docEditor.getSession().on("tokenizerUpdate", () => {
+                if (!alreadyFolded) {
+                    this.foldAll();
```

Comment:
1. I suggest calling `this.toggleAutoCollapse()` instead of `foldAll `since it will sync the _Toggle Collapse_ button
2. Do we want to do this also after clicking 'Refresh' button ?

Reply:
done